### ISSUE TITLE
Kaku input (doorbell, remote) won't be registered anymore after a 433 signal has been sent from Homey. 

### ISSUE THREAD

**Issue by bobenoni** [athombv/homey#831]
**What is your Homey version (Settings → System)?**

0.9.3

**What did you try to achieve?**

Commencing flows with 433 remote/doorbells as input. (But won't do so anymore if my homey has sent a 433 signal - reboot is required for remote/doorbell to function again. A new 433 signal sent from Homey makes remote/doorbell unfunctional again)

*Please include detailed steps how we can reproduce the issue*

Reboot - make led pulse with remote in flow - works everytime 
add a 433 signal to output (socket, doorbell - doesn't matter) - Flow works only once (without reboot).

**What was the result?**

433 signals can be sent from homey after "crash", but won't listen to remote/doorbell anymore.

...

**What did you expect as result?**
Nexa remotes/doorbells would work even after Homey has sent 433 signals
...

**Any other remarks**
 My brand is NEXA, not KAKU. But it pairs fine - and works as long as a 433 signal hasn't been sent from Homey.
...

**Comment by ReinierH** [comment]
Hi Bobenoni,

Could you provide me your system log? You send in a report on the settings->system->Send log

**Comment by bobenoni** [comment]
Sure,
CAE64C4C1E

Last thing I did now was replicating this issue.

**Comment by bobenoni** [comment]
Issue remains in 0.9.10

**Comment by ReinierH** [comment]
Unfortunately i was unable to reproduce this problem and had to patch fixes based on assumptions. Until yesterday, when i finally experienced the problem myself and found the probable cause. At my Homey this bug arises rarely, it looks like you consistently experience this problem, which makes it easy to verify if the current patch fixes the problem. Could you contact me  by mail?

**Comment by bobenoni** [comment]
sure

**Comment by sleutelsnet** [comment]
In 0.10.0, this issue still persists.
Battery powered 433 devices can be connected to homey, but homey cannot receive signals from them (or only receives one or two signals, and then goes mute for additional signals from the battery powered 433 device)
Has this bug not been squashed?

**Comment by bobenoni** [comment]
Improving in 0.10.1! Does not freeze on the first try like it used to, but Homey got hearing impaired on a couple of occations (missed a couple of button presses) and went totally deaf after ~20 button presses.

Testing flow with the test button still works, so I guess its the same type of crash

6ADDF82846

**Comment by bobenoni** [comment]
Installed 0.10.3 today, seems like this problem is fixed!
Good job!
